### ISSUE TITLE
Create an Alley APM package

### ISSUE THREAD

**Issue by kaitlinbolling** [alleyinteractive/accessibility-patterns#14]
### Description

An APM package should be created that allows developers to get up and running quickly.

### Use Case

An Alley developer should be able to run `apm install accessibility-patterns` to get a local environment set up quickly in order to easily contribute to the repository.
